pkg/v1/providers/ytt/03_customizations/02_avi/add_akoo.yaml

```diff
@@ -2,15 +2,15 @@
 #@ load("@ytt:json", "json")
 #@ load("static.lib.yaml", "crd","service")
 #@ load("rbac.lib.yaml", "role", "role_binding")
-#@ load("ako-operator-deployment.lib.yaml", "akoo_deployment", "ako_deployment_config", "avi_credentials", "avi_certificate")
+#@ load("ako-operator-deployment.lib.yaml", "akoo_deployment", "avi_credentials", "avi_certificate")
 
+#! render the ako-operator artifacts
 #@ if data.values.PROVIDER_TYPE == "vsphere" and data.values.TKG_CLUSTER_ROLE != "workload" and data.values.AVI_ENABLE and data.values.AVI_CONTROL_PLANE_HA_PROVIDER:
 --- #@ crd()
 --- #@ service()
 --- #@ role()
 --- #@ role_binding()
 --- #@ akoo_deployment()
---- #@ ako_deployment_config()
```

Comment:
Should also remove ako deployment config lib

Reply:
Could you clarify? Do you mean we don't need the `akoo_deployment` `avi_credentials` and `avi_certificate` here?